server.js

```diff
@@ -429,6 +436,18 @@ module.exports.initExpress = function () {
   app.use(require('./middlewares/authn')); // authentication, set res.locals.authn_user
   app.use('/pl/api', require('./middlewares/authnToken')); // authn for the API, set res.locals.authn_user
 
+  if (config.sentryDsn) {
+    // Attach user ID to Sentry events.
+    app.use((req, res, next) => {
+      if (res.locals.authn_user) {
+        Sentry.configureScope((scope) => {
+          scope.setUser({ id: res.locals.authn_user.user_id.toString() });
```

Comment:
I configured this to test out how Sentry works with user information, but I'm not sure if we actually want to land this. Under GDPR, I don't believe internal user IDs constitute PII, but we may want to avoid sending this to be on the safe side?

Reply:
I think this is fine and it's very helpful debugging context.

Can we add other things here, like `res.locals.response_id` and many of the other fields in [`logRequest`](https://github.com/PrairieLearn/PrairieLearn/blob/master/middlewares/logRequest.js) and [`logResponse`](https://github.com/PrairieLearn/PrairieLearn/blob/master/middlewares/logResponse.js)?

Reply:
Yes! I added some more context in 5fb320c8.